android/java/MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/views/NativeMapView.java

```diff
@@ -578,6 +585,8 @@ private native void nativeSetBearing(long nativeMapViewPtr, double degrees,
     private native void nativeSetSprite(long nativeMapViewPtr, String symbol,
                                         int width, int height, float scale, byte[] pixels);
 
+    private native void nativeSetVisibleCoordinateBounds(long mNativeMapViewPtr, LatLng[] coordinates, RectF padding, double direction, long duration);
```

Comment:
You reformatted the other lines to align with the bracket but not this long line.

Either remove the reformat above or make this new line consistent.